pyproject.toml

```diff
@@ -0,0 +1,61 @@
+[project]
+name = "meilisearch"
+description = "The python client for Meilisearch API."
+license = { text = "MIT" }
+requires-python = ">=3.7"
+authors = [
+  { name = "Charlotte Vermandel", email = "charlotte@meilisearch.com" },
+]
+keywords = [
+  "search",
+  "python",
+  "meilisearch",
+]
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: OS Independent",
+]
+dependencies = [
+    "requests",
+    "camel-converter[pydantic]",
+    "black",
+    "isort",
+]
+dynamic = ['version']
```

Comment:
Are you sure this will work for versioning?

Reply:
I didn't test it completly yet, I think we might have to add a ``[tool.setuptools.dynamic]`` section such as [here](https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html) and setup the version like this ``version = {attr = "my_package.__version__"}``

Reply:
Thanks for the input, I added it to the ``pyproject.toml``